miso-web/src/main/webapp/scripts/pool_ajax.js

```diff
@@ -509,21 +504,6 @@ Pool.ui = {
       "iDisplayLength": 25,
       "iDisplayStart": 0,
       "sDom": '<l<"#toolbar">f>r<t<"fg-toolbar ui-widget-header ui-corner-bl ui-corner-br ui-helper-clearfix"ip>',
-      "fnRowCallback": function(nRow, aData, iDisplayIndex, iDisplayIndexFull) {
-        Fluxion.doAjax(
-          'poolControllerHelperService',
-          'checkAverageInsertSizeByPoolId',
-          {
-            'poolId':aData.id,
-            'url':ajaxurl
-          },
-          {
-            'doOnSuccess': function(json) {
-              jQuery('td:eq(5)', nRow).html(json.response);
-            }
-          }
-        );
-      },
       "aaSorting": [
         [(Sample.detailedSample ? 7 : 0) , "desc"] // NB: this must get updated when adding new columns
```

Comment:
Fixed. I've added the code that I thought I added before that selects a column by priority.